docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/focus-entity.md

```diff
@@ -14,4 +14,10 @@ status: draft
 
 # What is a Focus entity?
 
-If you navigate to the Genome browser from an entity, such as a gene or variant, either by searching for that entity or from the Entity viewer, that entity is your Focus entity. The Focus entity is highlighted in the browser image by red dotted lines. The name and/or identifier and coordinates of the Focus entity are displayed in the Navigation bar. Details of the Focus entity are displayed at the top of the Track panel.
+The gene, variant (or any other entity) you have searched for will be front and centre in the Genome browser view and is known as the Focus entity. 
+
+The boundaries of Focus entity are highlighted in the browser by red dotted lines. 
+
+The name and/or identifier and coordinates of the Focus entity are displayed in the Navigation bar. 
+
+Details of the Focus entity are displayed at the top of the right-hand Track panel.
```

Comment:
Add a detail to make it easier to understand this ? How about:
'Details of the Focus entity, for example the symbol for a gene and the number of its transcripts, are displayed at the top right...'

Reply:
done